src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { Todo } from "../../types/Todo";
+import { UserInfo } from "../UserInfo";
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const userObj = usersFromServer.find(user => user.id === todo.userId);
```

Comment:
Ensure that `usersFromServer` is always available and correctly formatted. If there's a possibility that the data might not be loaded or could be malformed, consider adding error handling or a fallback mechanism.

Reply:
Done